### PR TITLE
Add BOBSL, ISL-HS, Sign-BD datasets to the datasets table and references.bi

### LINKED ISSUE #27: Document several datasets noticed during my reading:  SignBD, Irish Sign Language, ArASL2018, BOBSL
Been sifting through the literature, ran into some datasets.

SignBD, Irish Sign Language, ArASL2018, and esp BOBSL which seems quite notable/large could be added to the documentation 

Would just need to go to https://github.com/sign-language-processing/sign-language-processing.github.io/tree/master/src/datasets and add JSON for each, then submit a pull request.

I should be able to take this on quick.

### PR DESCRIPTION
**Pull request by cleong110**
Closes #27 if accepted

### CODE REVIEW

File: src/datasets/BOBSL.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "video:RGB",
+      "text:English"
+    ],
+    "language": "British Sign Language (BSL)",
+    "#items": 2281,
+    "#samples": "1.2M Sentences",
+    "#signers": 37,
+    "license": "non-commercial authorized academics",
+    "licenseUrl": "https://www.bbc.co.uk/rd/projects/extol-dataset",
+    "contact": "Samuel Albanie albanie[AT]robots.ox.ac.uk"
```
Comment (AmitMY) [inline src/datasets/BOBSL.json:18]: i’d change the contact to a real email address

File: src/references.bib [hunk 2]
```diff
@@ -2076,4 +2076,35 @@ @inproceedings{muller-etal-2023-considerations
...
+  abstract = {Recently, sign language researchers have turned to sign language interpreted TV broadcasts, comprising (i) a video of continuous signing and (ii) subtitles corresponding to the audio content, as a readily available and large-scale source of training data. One key challenge in the usability of such data is the lack of sign annotations. Previous work exploiting such weakly-aligned data only found sparse correspondences between keywords in the subtitle and individual signs. In this work, we propose a simple, scalable framework to vastly increase the density of automatic annotations. Our contributions are the following: (1)~we significantly improve previous annotation methods by making use of synonyms and subtitle-signing alignment; (2)~we show the value of pseudo-labelling from a sign recognition model as a way of sign spotting; (3)~we propose a novel approach for increasing our annotations of known and unknown classes based on in-domain exemplars; (4)~on the BOBSL BSL sign language corpus, we increase the number of confident automatic annotations from 670K to 5M. We make these annotations publicly available to support the sign language research community.},
```
Comment (AmitMY) [inline src/references.bib:2091]: i think for making this file not huge, we decided to not include abstracts

File: src/datasets/ISL-HS.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "pub": {
+      "name": "ISL-HS",
+      "year": 2017,
+      "publication": "dataset:oliveiraDatasetIrishSign2017",
```
Comment (AmitMY) [inline src/datasets/ISL-HS.json:5]: missing citation from bib

File: src/datasets/BOBSL.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      "name": "BOBSL",
+      "year": 2022,
+      "publication": "dataset:momeniAutomaticDenseAnnotation2022",
+      "url": "https://www.robots.ox.ac.uk/~vgg/data/bobsl/"
+    },
+    "features": [
+      "video:RGB",
+      "text:English"
+    ],
+    "language": "British Sign Language (BSL)",
```
Comment (AmitMY) [inline src/datasets/BOBSL.json:12]: Should be "British"

File: src/datasets/ISL-HS.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      "url": "https://github.com/marlondcu/ISL"
+    },
+    "features": [
+      "video:RGB",
+      "gloss:ISL-HandShapes"
+    ],
+    "language": "Irish Sign Language (ISL)",
```
Comment (AmitMY) [inline src/datasets/ISL-HS.json:12]: should be "Irish"

File: src/datasets/ISL-HS.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    "#items": 23,
+    "#samples": "468 videos available, 58,114 images extracted to show 23 handshapes",
```
Comment (AmitMY) [inline src/datasets/ISL-HS.json:14]: While here I'd say that more details are better, please consider how this is displayed:
I think less information should be present, but if you still wanted to include the entire text, I'd say narrow it to
"468 videos → 58,114 images → 23 handshapes"
image

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/datasets/ISL-HS.json (hunk 3 above)
Comment (cleong110) [reply src/datasets/ISL-HS.json:5]: So it is, fixing that

File: src/datasets/ISL-HS.json (hunk 5 above)
Comment (cleong110) [reply src/datasets/ISL-HS.json]: fixed

File: src/datasets/BOBSL.json (hunk 4 above)
Comment (cleong110) [reply src/datasets/BOBSL.json]: fixed

File: src/datasets/BOBSL.json (hunk 1 above)
Comment (cleong110) [reply src/datasets/BOBSL.json]: Ah yes, copy-pasted that from the project website, good suggestion

**Review by AmitMY — APPROVED** [review]

**Pull request merged**